example/src/main/java/org/nervos/ckb/ACPTransactionExample.java

```diff
@@ -0,0 +1,187 @@
+import org.nervos.ckb.type.transaction.Transaction;
+import org.nervos.ckb.utils.Numeric;
+import org.nervos.ckb.utils.Utils;
+import org.nervos.ckb.utils.address.AddressGenerator;
+import org.nervos.ckb.utils.address.AddressParser;
+
+/** Copyright © 2021 Nervos Foundation. All rights reserved. */
+// ACP RFC:
+// https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0026-anyone-can-pay/0026-anyone-can-pay.md
+public class ACPTransactionExample {
+  // ACP_CKB_MINIMUM is set to 9, which means in each transaction, one must at least transfers 10^9
+  // shannons, or 10 CKBytes into the anyone-can-pay cel
+  private static final String ACP_CKB_MINIMUM = "09";
+
+  private static Api api;
+  private static CkbIndexerApi ckbIndexerApi;
+
+  private static final List<String> SendPrivateKeys =
+      Collections.singletonList("d00c06bfd800d27397002dca6fb0993d5ba6399b4238b2f29ee9deb97593d2bc");
+  private static final List<String> SendAddresses =
+      Collections.singletonList("ckt1qyqvsv5240xeh85wvnau2eky8pwrhh4jr8ts8vyj37");
+  private static final List<String> ReceiveAddresses =
+      Collections.singletonList("ckt1qyqtnz38fht9nvmrfdeunrhdtp29n0gagkps4duhek");
+  private static String receiverAcpAddress;
+
+  static {
+    api = new Api(NODE_URL, false);
+    ckbIndexerApi = new CkbIndexerApi(CKB_INDEXER_URL, true);
+
+    Script receiverScript = AddressParser.parse(ReceiveAddresses.get(0)).script;
+    receiverScript.codeHash = ACP_CODE_HASH;
+    receiverScript.args = receiverScript.args + ACP_CKB_MINIMUM;
+    receiverAcpAddress = AddressGenerator.generate(Network.TESTNET, receiverScript);
+  }
+
+  public static void main(String[] args) throws Exception {
+    System.out.println(
+        "Before transferring, first sender's balance: "
+            + getBalance(SendAddresses.get(0)).divide(UnitCKB).toString(10)
+            + " CKB");
+
+    String acpHash = createACPCell();
+    System.out.println("Create acp cell tx hash: " + acpHash);
+
+    // waiting transaction into block, sometimes you should wait more seconds
+    Thread.sleep(30000);
+    System.out.println("Transfer SUDT tx hash: " + transfer(new OutPoint(acpHash, "0x0")));
+  }
+
+  private static BigInteger getBalance(String address) throws IOException {
+    return new IndexerCollector(api, ckbIndexerApi).getCapacity(address);
+  }
+
+  private static String createACPCell() throws IOException {
+    List<ScriptGroupWithPrivateKeys> scriptGroupWithPrivateKeysList = new ArrayList<>();
+
+    TransactionBuilder txBuilder = new TransactionBuilder(api);
+    IndexerCollector txUtils = new IndexerCollector(api, ckbIndexerApi);
+
+    List<Receiver> receivers =
+        Collections.singletonList(new Receiver(receiverAcpAddress, Utils.ckbToShannon(200)));
+    List<CellOutput> cellOutputs = txUtils.generateOutputs(receivers, SendAddresses.get(0));
+    txBuilder.addOutputs(cellOutputs);
+
+    txBuilder.setOutputsData(Arrays.asList("0x", "0x"));
```

Comment:
There is no need for manual assignment, right? https://github.com/nervosnetwork/ckb-sdk-java/blob/11531e0f4b94de898553f8e01e0c7f278d914c65/ckb/src/main/java/org/nervos/ckb/transaction/TransactionBuilder.java#L94

Reply:
No. For a simple transaction, the developer maybe does not care about the outputs_data, and he could not set outputs_data.

Reply:
Why do you manually set `outputs_data ` here?

Reply:
If the developer forgets to set the `outputsData`, I will help him to fix it